src/docs/reference/modules/Roles/README.md

```diff
@@ -4,17 +4,20 @@ Enabling the `OrchardCore.Roles` module will allow you to manage the user roles.
 
 ## Predefined Roles
 
-Orchard Core come up with the following predefined permission stereotypes:
+Orchard Core defines the following predefined permission stereotypes:
 
 | Name | Description |
 | --- | --- |
-| `Administrator` | Contains all the administrator users. |
-| `Anonymous` | Contains all the non authenticated users. |
-| `Authenticated` | Contains all the authenticated users. |
-| `Author` | Contains all the users who have the ability to author contents. |
-| `Contributor` | Contains all the users who have the ability to contribute to the contents. |
-| `Editor` | Contains all the users who have the ability to edit the contents. |
-| `Moderator` | Contains all the users who have the ability to moderate the contents. |
+| `Administrator` | A **system role** that grants all permissions to the assigned users. |
+| `Anonymous` | A **system role** representing all non-authenticated users. |
+| `Authenticated` | A **system role** representing all authenticated users. |
+| `Author` | Grants users the ability to create content. |
+| `Contributor` | Grants users the ability to contribute content. |
+| `Editor` | Grants users the ability to edit existing content. |
+| `Moderator` | Grants users the ability to moderate content. |
```

Comment:
The recipes now don't have any role descriptions. What they had wasn't useful, but this would be. Yes, people can look up the docs, but having these right there on the Roles screen would be even better.

Reply:
Yes, if we need to, we can update the description of each role in the recipe with something other than the name of the role. For now, I just removed the redundant text.

Reply:
I'd use these exact ones.

Reply:
Sorry spent way more time on this PR than I wanted to. I don't have time to update all these recipes with text and ensure I don't copy to the wrong role. Feel free to direct commit to the PR if you like to update the descriptions.

Reply:
Sure, done it.

Reply:
Awesome! Thank you